.gitignore

```diff
@@ -16,11 +16,17 @@ x64/
 [Bb]in/
 [Oo]bj/
 
+[Tt]ools/
```

Comment:
Ok, that's doable.  What would you expect for "clean binaries" to do then?  Remove just the Release and Debug subdirs?